charts/assemble-backstage/templates/postgres-pvc.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.postgres.persistent }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name:  {{ include "assemble-backstage.fullname" . }}-postgres
```

Comment:
The StatefulSet requests the following PVC:
```
      volumes:
      - name: postgresdb
        persistentVolumeClaim:
          claimName: {{ include "assemble-backstage.fullname" . }}-postgresql
```
So the name of this PVC should end with `-postgresql` instead of `-postgres`